modules/govuk_elasticsearch/spec/classes/govuk_elasticsearch_spec.rb

```diff
@@ -23,11 +23,11 @@
 
   describe '#manage_repo' do
     let(:params) {{
-      :version => '1.7.0',
+      :version => '1.4.3',
```

Comment:
These are test values for rspec and are only used when running spec tests, they are not visible, and will not be used, on actual puppet runs.